### ISSUE TITLE
Home Directory showing empty

### ISSUE THREAD

**Issue by satishvis** [gokcehan/lf#769]
I am unable to see my files in the home directory. 
It is showing empty.
This is probably if I mount a fuse drive like gdrive in my case.
On unmounting it is back to normal

OS: Arch Linux
Problem: rclone mount of gdrive does not allow reading of files in home directory

**Comment by kmarius** [comment]
I briefly tried it with rclone and can not reproduce. I have however observed it with hanging `sshfs` mounts where a single call to `stat` or `chdir` would block and stall the application (until it times out or is forcefully deconnected). Some questions:

1. Does it happen with no config, i.e. `lf -config /dev/null`?
2. Does it show `empty` or `loading`?
3. Are you still able to navigate or quit?
4. Do the logs show anything (under `/tmp/lf.*.log`)?
5. Does it work in e.g. `ranger`? (all terminal managers behave the same regarding the `sshfs` issue)

**Comment by satishvis** [comment]
Not able to reproduce in my main machine. 
`lf` is working fine inspite of the rclone mount.
It is even reading the gdrive fine, albeit slow.
The issue propped up in a new installation of Arch in my usb. 
Will check and come back

**Comment by kmarius** [comment]
@satishvis You don't need to answer my questions, the bug is essentially the same as in #741. Lf encounters an error while trying to load the file (the mountpoint) when loading the home directory. It just bails out and returns an empty file list (likely because the mountpoint is is the first file in your home directory).

I can reproduce it as follows, with `rclone` set up, the remote name, also a gdrive, `remote` and mountpoint `a`:
1. `rclone mount remote: a`
2. start another shell, navigate into `a`
3. stop `rclone` with `ctrl-c`, unmount fails with `Device or resource busy`
4. start `lf`

Log: 
```
2022/02/23 21:54:48 reading directory: lstat /home/marius/a: transport endpoint is not connected
```

**Comment by gokcehan** [comment]
I have pushed a fix for #741 which likely fixes this issue as well. Closing this issue now. Feel free to reopen if the issue persists.
